<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>worktext</class>
 <widget class="QDialog" name="worktext">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>981</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Добавиит текст</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="leftTE"/>
    </item>
    <item>
     <widget class="QPushButton" name="swapButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Заменить текст</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="rightTE"/>
    </item>
    <item>
     <widget class="QPushButton" name="htmlButton">
      <property name="text">
       <string>HTML</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
